Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -292,23 +292,27 @@ private void RunMatchExpression(FlexMatch binary64DecodedMatch, AnalyzeContext c
                             // The validator returned a bad value.
                             // TODO: we should log this condition
                             // and then continue processing.
+                            level = FailureLevel.Error;
                             break;
                         }
 
                         case Validation.Valid:
                         {
+                            level = FailureLevel.Error;
```

Comment:
>level [](start = 28, length = 5)

If we conclusively validate something, it's important! So we make it an error.

Reply:
yeah, we should think how we will log, so we can decide in the future based on data.

Reply:
but, ur decision is perfect based on our knowledge so far